client/src/components/SliderContainer/SliderContainer.js

```diff
@@ -0,0 +1,80 @@
+import React, { Component } from "react";
+import "./SliderContainer.css";
+import PropTypes from "prop-types";
+import hawaii from "../../../../assets/images/hawaii.png";
+import southernCaribbean from "../../../../assets/images/southernCaribbean.png";
+import easternCaribbean from "../../../../assets/images/easternCaribbean.png";
+import hawaiiTrees from "../../../../assets/images/hawaii-trees.png";
+import westernCaribbean from "../../../../assets/images/westernCaribbean.png";
+import CarouselDots from "../CarouselDots/CarouselDots.js";
+
+class SliderContainer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      image_index: 0
```

Comment:
`imageIndex`! This is JS not Python!

Reply:
ah. cool cool cool. I bet I'm gonna make this mistake a lot. fixed this one and the start/end dates